source/index.ts

```diff
@@ -1,7 +1,9 @@
-import packageJson = require('../package.json');
+import { readFileSync } from 'fs';
 import create from './create';
 import {Defaults} from './utils/types.js';
 
+const packageJson = JSON.parse(readFileSync('./../package.json', { encoding: 'utf8' }));
```

Comment:
You have to use `__dirname` and path.join here.

Reply:
okay, done 😊